WooCommerce/Classes/ViewRelated/Products/Edit Product/Edit Downloads/Download List/ProductDownloadDnD.swift

```diff
@@ -0,0 +1,75 @@
+import Foundation
+import CoreServices
+import Yosemite
+
+/// A wrapper around ProductDownload, to make it compatible in using as Drag and Drop data source in an table view. Represents a ProductDownload entity.
```

Comment:
```suggestion
/// A wrapper around `ProductDownload, to make it compatible in using as Drag and Drop data source in a table view. Represents a `ProductDownload` entity.
```

It will be also useful to describe why we are not using directly `ProductDownload`.

Reply:
Yeap described the reasoning